events/story_cycles/wc_story_cycle_horde_invasion_events.txt

```diff
@@ -98,13 +86,50 @@ WCHOI.3 = {
 			}
 		}
 		name = WCHOI_3_OPT_A
+
+		if = {
+			limit = {
+				NOT = { root = scope:warchief }
+				highest_held_title_tier < scope:warchief.highest_held_title_tier
```

Comment:
You should check if I'm already a vassal.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114561832-fda20c00-9c7e-11eb-83f0-179ed23fee72.png)

</details>